softdevice_controller/include/sdc_hci_vs.h

```diff
@@ -809,6 +832,35 @@ uint8_t sdc_hci_cmd_vs_coex_scan_mode_config(const sdc_hci_cmd_vs_coex_scan_mode
  */
 uint8_t sdc_hci_cmd_vs_coex_priority_config(const sdc_hci_cmd_vs_coex_priority_config_t * p_params);
 
+/** @brief Disable peripheral latency.
+ *
+ * This vendor specific command is used to disable/enable peripheral latency.
+ * Three modes of disable/enable are supported.
+ *
+ * In enable mode, the peripheral will wake up every time it has data to send,
+ * and/or every peripheral latency number of connection events.
+ *
+ * In disable mode, the peripheral will wake up on every connection event
+ * regardless of the requested peripheral latency. This option consumes the most power.
+ *
+ * In wait_for_ack mode, the peripheral will wake up on every connection event if
+ * it has not received an ACK from the master for at least peripheral latency events.
+ * This configuration may increase the power consumption in environments with a lot of radio
+ * activity.
+ *
+ * @note Shall only be called on peripheral links.
+ *
+ * Event(s) generated (unless masked away):
+ * When the command has completed, an HCI_Command_Complete event shall be generated.
+ *
+ * @param[in]  p_params Input parameters.
+ *
+ * @retval 0 if success.
+ * @return Returns value between 0x01-0xFF in case of error.
+ *         See Vol 2, Part D, Error for a list of error codes and descriptions.
+ */
+uint8_t sdc_hci_cmd_vs_peripheral_latency_disable(const sdc_hci_cmd_vs_peripheral_latency_disable_t * p_params);
```

Comment:
why is the command called latency disable, when it can be used to enable peripheral latency?

Reply:
Renamed to sdc_hci_cmd_vs_peripheral_latency_mode_set